src/markdown-pages/add-ons/aws.md

```diff
@@ -22,6 +22,6 @@ flags-table
 
 ## Requirements and Limitations
 
-* [Does this work for air gap installations on kURL clusters?]
-* [Do you need a specific port available?]
-* [Other?]
+* When used, a volume provisioner add-on does not need to be included in the kURL specification as [AWS EBS](https://aws.amazon.com/ebs/) will be used.
```

Comment:
I wouldn't call this a limitation

Reply:
Agree, though felt like it was more or less a requirement or something that should be mentioned. Maybe requirements and limitations isn't the best place. @paigecalvert wdyt?

Reply:
Oh good call--I created a separate section for this info with my latest commit. See whatcha think https://github.com/replicatedhq/kurl.sh/pull/784/commits/1aa2043e61be3cc190ae0dac5703df6626b09da5